addons/website_sale_delivery/models/sale_order.py

```diff
@@ -1,121 +1,83 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 import logging
 
-from openerp.osv import orm, fields
-from openerp import SUPERUSER_ID
-from openerp.addons import decimal_precision
-from openerp.exceptions import ValidationError
-from openerp.tools.translate import _
-
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
 
 _logger = logging.getLogger(__name__)
 
 
-class delivery_carrier(orm.Model):
-    _name = 'delivery.carrier'
-    _inherit = ['delivery.carrier', 'website.published.mixin']
-
-    _columns = {
-        'website_description': fields.related('product_id', 'description_sale', type="text", string='Description for Online Quotations'),
-    }
-    _defaults = {
-        'website_published': False
```

Comment:
But this is an override. When you see commits like bcabe788ee7fa30be5330120a97be103c52b5d3b, you understand that this is not that obvious.